<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="RadioBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>201</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>RadioBox</string>
   </property>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>112</width>
      <height>141</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="radio_1">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_2">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_3">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="InputBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Box</string>
   </property>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>70</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QCheckBox" name="Input1">
     <property name="text">
      <string>Input1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Input2">
     <property name="text">
      <string>Input2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Input3">
     <property name="text">
      <string>Input3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Input4">
     <property name="text">
      <string>Input4</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="OutputBox">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>151</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Box</string>
   </property>
   <widget class="QSplitter" name="splitter_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>111</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QCheckBox" name="Output1">
     <property name="text">
      <string>Output1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Output2">
     <property name="text">
      <string>Output2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Output3">
     <property name="text">
      <string>Output3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="Output4">
     <property name="text">
      <string>Output4</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_c">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>837</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>840</y>
     <width>20</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_s">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>840</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>311</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_enter">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_font_ubuntu">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>470</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ubuntu</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_font_nanumgothic">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>490</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>nanum</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_colour_red">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>520</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_colour_green">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>540</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_colour_blue">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>560</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_font_size">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>620</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_font_size">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>600</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="btn_1">
    <property name="text">
     <string>Button 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="text">
     <string>Button 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="text">
     <string>Button 3</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_pop_name">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>680</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pop_season">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>700</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Season</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pop_color">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>720</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pop_font">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>680</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_pop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pop_file">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>700</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pop">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>720</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
